### ISSUE TITLE
Crash just after showing window

### ISSUE THREAD

**Issue by jegjessing** [elementary/mail#189]
Pantheon-mail suddenly crashes before I can get to interact with it.

I'm not sure how to debug this, but would be more than happy to supply any information you need.

**Comment by yavinfour** [comment]
I'm also having this same issue. I attempt to open Mail and the window shows and then immediately closes. I'm happy to provide log information if someone can point me to where to look.

**Comment by yavinfour** [comment]
Opening it from Terminal produced this:

```
(pantheon-mail:5106): Gtk-WARNING **: drawing failure for widget 'StylishWebView': invalid value (typically too big) for the size of the input (surface, pattern, etc.)

(pantheon-mail:5106): Gtk-WARNING **: drawing failure for widget 'GtkGrid': invalid value (typically too big) for the size of the input (surface, pattern, etc.)

(pantheon-mail:5106): Gtk-WARNING **: drawing failure for widget 'GtkGrid': invalid value (typically too big) for the size of the input (surface, pattern, etc.)

(pantheon-mail:5106): Gtk-WARNING **: drawing failure for widget 'GtkRevealer': invalid value (typically too big) for the size of the input (surface, pattern, etc.)

(pantheon-mail:5106): Gtk-WARNING **: drawing failure for widget 'GtkGrid': invalid value (typically too big) for the size of the input (surface, pattern, etc.)

(pantheon-mail:5106): Gtk-WARNING **: drawing failure for widget 'ConversationWidget': invalid value (typically too big) for the size of the input (surface, pattern, etc.)

(pantheon-mail:5106): Gtk-WARNING **: drawing failure for widget 'GtkListBox': invalid value (typically too big) for the size of the input (surface, pattern, etc.)

(pantheon-mail:5106): Gdk-ERROR **: The program 'pantheon-mail' received an X Window System error.
This probably reflects a bug in the program.
The error was 'BadAlloc (insufficient resources for operation)'.
  (Details: serial 3115 error_code 11 request_code 53 (core protocol) minor_code 0)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the GDK_SYNCHRONIZE environment
   variable to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
Trace/breakpoint trap (core dumped)
```

**Comment by yavinfour** [comment]
If I tapped the down button to move to a different message as the window appears, Mail does not crash. From the log above it appears that, perhaps, the email message at the top of my Inbox is too long/big for the UI to draw.

**Comment by kdekooter** [comment]
I think this is a duplicate of #33

**Comment by mdh34** [comment]
Closed as a duplicate of #27
